Bottom: fcc922b3867725a607261c11f3e65da207856e70
Top:    e1f4fd4cb6d9028c4e97208e717a82f10af9f289
Author: Ian Kent <ikent@redhat.com>
Date:   2013-04-03 16:28:52 +0800

Refresh of debug.patch

---

diff --git a/daemon/automount.c b/daemon/automount.c
index 252a49e..462bba6 100644
--- a/daemon/automount.c
+++ b/daemon/automount.c
@@ -1344,11 +1344,11 @@ static void *statemachine(void *arg)
 			master_mutex_lock();
 			if (list_empty(&master_list->completed)) {
 				if (!list_empty(&master_list->mounts)) {
-					debug(LOGOPT_ANY, "list empty completed and mounts");
+					debug(LOGOPT_ANY, "list empty completed and mounts not empty");
 					master_mutex_unlock();
 					return NULL;
 				}
-				debug(LOGOPT_ANY, "list empty completed, do nothing");
+				debug(LOGOPT_ANY, "list empty completed and mounts, shutdown");
 			} else {
 				if (master_done(master_list)) {
 					debug(LOGOPT_ANY, "master_done returns true");
